modules/ob-enabling-builds-in-a-network-restricted-environment-using-the-source-to-image-build.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * builds/uninstalling-openshift-builds.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id='ob-creating-source-to-image-build-in-a-network-restricted-environment_{context}']
+= Creating source-to-image build in a network restricted environment
+
+You can create a `source-to-image` build in a network restricted environment by mirroring the images required by `source-to-image` BuildStrategy. 
+
+.Prerequisites
+
+* Your cluster can connect and interact with the git source that you will use to create the source-to-image build.
+* You have the builder-image required to create the `source-to-image` build in your local registry. If you do not have the builder-image in the local registry, mirror the source image. For more information on mirroring images for network restricted environment, see: link:https://docs.openshift.com/container-platform/4.17/disconnected/mirroring/installing-mirroring-installation-images.html[Mirroring images for a disconnected installation by using the oc adm command].
```

Comment:
This link needs to be an xref. I previously linked the Contributing Guide info on this, but please feel free to ping me if you have questions on this or need help :)